src/Miningcore/Payments/PaymentSchemes/PPSPaymentScheme.cs

```diff
@@ -329,11 +296,7 @@ private async Task<decimal> CalculateBlockData(decimal blockRewardInEth, double
             }
 
             /* update the value per hash in the pool payment processing config based on latest calculation */
-            Decimal valPerHash = 0;
```

Comment:
this was actually put in to account for a potential divideByZero error.  sumDifficulty can be zero.  can you add this small block back in?

Reply:
Fixed